Sources/main.cpp

```diff
@@ -329,8 +329,6 @@ static int PlatformMain(int argc, char **argv) {
 
     thread.detach();
   } while (true);
-
-  return EXIT_SUCCESS;
 }
```

Comment:
Add an unreachable annotation?

Reply:
I don't think having an unreachable here would be of any use. These annotations are meant to help the compiler in general. Thoughts?